models/members.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const logger = require('../utils/logger')
+const firestore = require('../utils/firestore')
+const userModel = firestore.collection('users')
+
+/**
+ * Fetches the data about our members
+ * @param query { Object }: Filter for members data
+ * @return {Promise<userModel|Array>}
+ */
+const fetchMembers = async (query) => {
+  try {
+    const snapshot = await userModel.get()
```

Comment:
@swarajpure , can you check if there is any way to directly query only those documents with `isMember` as `true`?

Reply:
Done. Thank you!